resources/views/components/movie-card.blade.php

```diff
@@ -0,0 +1,29 @@
+@props([
+    "title" => "",
+    "year" => "",
+    "length" => "",
+    "description" => "",
+    "movie" => [],
+])
+
+@php
+    $image = $movie["image"] ?? asset("images/default.jpg");
+@endphp
+
+<x-base-card class="m-4 w-full h-auto">
+    <div class="flex gap-4">
+        <img class="w-16 h-24 object-cover rounded-lg" src="{{ $image }}" alt="{{ $title }}">
```

Comment:
```suggestion
        <img class="w-16 h-24 object-cover" src="{{ $image }}" alt="{{ $title }}">
```
Your example photos are not showing any rounding. Figma is not saying img should have rounding either

Reply:
my cat stepped on the keyboard

Reply:
its ok, i want cat